ci/README.md

```diff
@@ -1,14 +1,19 @@
 # Configuring the CI Environment
 
-We use several Google Cloud Platform services to perform integration tests.
-These notes describe how to setup a project to run these integration tests.
-The main audience for these notes are developers working to improve the
-Functions Framework for C++. They assume the reader is familiar with GCP and
-with the Google Cloud SDK command-line tool.
+These notes describe how to set up a GCP project to run these Functions Framework for C++ integration tests. The main
+audience for these notes are developers of the Functions Framework.
+
+- If you are looking for examples on how to **use** the Functions Framework for C++, the
+  [How-to Guides](/examples/howto-guides.md) should help you get started.
+- If you want to compile and test the examples after making changest to the framework, the read
```

Comment:
s/changest/changes/
s/the read/then read/

Reply:
Done.